cmd/notation/internal/display/handler.go

```diff
@@ -21,12 +25,13 @@ import (
 	"github.com/notaryproject/notation/cmd/notation/internal/display/metadata"
 	"github.com/notaryproject/notation/cmd/notation/internal/display/metadata/json"
 	"github.com/notaryproject/notation/cmd/notation/internal/display/metadata/tree"
+	"github.com/notaryproject/notation/cmd/notation/internal/display/output"
 	"github.com/notaryproject/notation/cmd/notation/internal/option"
-	"github.com/notaryproject/notation/cmd/notation/internal/output"
 )
 
-// NewInpsectHandler creates a new InspectHandler based on the output format.
-func NewInpsectHandler(printer *output.Printer, format option.Format) (metadata.InspectHandler, error) {
+// NewMetadataInpsectHandler creates a new InspectHandler based on the output
+// format.
+func NewMetadataInpsectHandler(printer *output.Printer, format option.Format) (metadata.InspectHandler, error) {
```

Comment:
Let's call it `NewInpsectHandler`.

In the package `display`, the function name is intended to be
```go
func New{{.CommandName}}Handler({{.parameters}}) (status.{{.CommandName}}Handler, metadata.{{.CommandName}}Handler, error)
```
although we dont have status support yet.

Reply:
Updated to `NewInpsectHandler`